commandapi-core/src/main/java/dev/jorel/commandapi/executors/CommandArguments.java

```diff
@@ -443,4 +445,55 @@ public <T> Optional<T> getOptionalUnchecked(String nodeName) {
 		return (Optional<T>) getOptional(nodeName);
 	}
 
+	/**
+	 * Returns an argument purely based on its CommandAPI representation. This also attempts to directly cast the argument to the type represented by {@link dev.jorel.commandapi.arguments.AbstractArgument#getPrimitiveType()}
+	 *
+	 * @param argumentType The argument instance used to create the argument
+	 * @return The argument represented by the CommandAPI argument, or null if the argument's clas cannot be cast to the type represented by {@link dev.jorel.commandapi.arguments.AbstractArgument#getPrimitiveType()}
```

Comment:
Typo: `clas`. Minor thing, but that's going to sit there forever otherwise :smile:

Reply:
Thanks xD